cmd/ctr/shim.go

```diff
@@ -10,6 +10,7 @@ import (
 	"os"
 	"strconv"
 	"time"
+	"errors"
```

Comment:
I think this should be `github.com/pkg/errors` so that we can use `Wrapf()`, but it can be later.